<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaHome</class>
 <widget class="QWidget" name="VistaHome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>576</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestore Centro Sportivo PRO</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">image: url(:/immagini/sfondo.jpg);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(69, 162, 0);button</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="1" column="0">
                <widget class="QListView" name="listViewPrenotazioni">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Le tue prenotazioni:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string>Disdici prenotazione</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QWidget" name="widget_5" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(69, 162, 0);</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="4" column="0">
             <widget class="QLineEdit" name="labelEmail"/>
            </item>
            <item row="6" column="0">
             <widget class="QLineEdit" name="labelAbbonamento"/>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="labelTelefono"/>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="labelCF"/>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="labelNomeCognome"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>IL TUO PROFILO</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="labelIndirizzo"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
